Makefile

```diff
@@ -151,6 +152,8 @@ ifneq ("$(ARCH)","arm")
 with_rdpclient := yes
 RDPCLIENT_MESSAGE := "with Windows RDP client"
 RDPCLIENT_TAG := desktop_access_rdp
+DIRECTORY_SHARING_TAG := directory_sharing
```

Comment:
Is there a point where RPD and directory sharing diverge? If they are meant to be used together, should we combine these with the RPD variables and simplify a bit?

Reply:
Technically its possible to build with the rdp client enabled but directory sharing disabled. I'm not sure there's a reason to continue to keep them separate.

cc @zmb3

Reply:
Yeah, good point. The build tag was just to ensure during dev we didn't activate half-completed code. +1 for removing it entirely.